src/pages/Contact.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import TextField from '@material-ui/core/TextField';
+import TextareaAutosize from '@material-ui/core/TextareaAutosize';
+import Button from '@material-ui/core/Button';
+import { Link } from 'react-router-dom';
+
+function Contact() {
+  return (
+    <div>
```

Comment:
Instead of a `<div>` have you looked into using a [`<Container>`](https://material-ui.com/components/container/#container) for these pages?